### PR TITLE
Feature: Add #5 episod of UnderJS to shows collection

### PR DESCRIPTION
**Pull request by ovr**

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -596,14 +596,28 @@
...
+        },
         {
             "show": "Frontend Weekend",
             "episodes": [
                 {
                     "num": "25",
                     "date": "2017-10-30",
                     "title": "Роман Дворнов о том, как создавать Open Source и не перегореть",
-                    "url": {
+                    "media": {
```
Comment (ovr) [inline data.json]: After small research realize that it should be `media`, because `url` is `string` type

**Comment by lahmatiy** [comment]
Thank you a lot!
Could you please run `npm run build` as well?

**Comment by lahmatiy** [comment]
Or make the same changes in docs/data.json

**Comment by lahmatiy** [comment]
Thank you 🙏

**Pull request merged**